epam_tr/src/main/java/com/epam/tr/dto/FileDto.java

```diff
@@ -4,17 +4,13 @@
 
 public class FileDto {
```

Comment:
OK, I reviewed whole the change.
I see you have FileRequestDto and FileDto now. It seems to be weird a little bit.
Earlier when you didn't have a FileRequestDto it was ok becase FileDto was like super smart dto. This dto acted as a request and a File.
But now, I guess, it makes sense to rename FileDto to FileSystemObject again. Because this file is responsible only for being system file, correct? We reach a single resposibility finally.

Reply:
Hmmm, I took a look once again... Actually you retrun FileDto from a controller, then it isn't I excpected, so do not rename yet.